4_1_Functions_and_geometry.ts

```diff
@@ -0,0 +1,41 @@
+// Jaime Restrepo
+// 4. Functions.
+// Exercise 1. Functions and geometry.
+
+import * as promptSync from "prompt-sync";
+const prompt = promptSync();
+
+
+console.log("**************************************************");
+console.log("Starting");
+console.log("Functions and geometry.");
+console.log("*******************************");
+
+// Named function:
+function circleArea(radius: number): number {
+    return 3.1416 * (radius ** 2);
+}
+
+// Anonymous function to calculate the perimeter of a rectangle:
+const rectanglePerimeter = function(length: number, height: number): number {
+    return 2 * length + 2 * height;
+}
+
+// Arrow function
+const cubeVolume = (sideLength: number): number => {
+    return sideLength ** 3;
+}
```

Comment:
Since this one is only returning one expression, it could be re-written as:
```suggestion
// Arrow function
const cubeVolume = (sideLength: number): number => sideLength ** 3;
```